containers/ruby-consul-terraform/render_configs.rb

```diff
@@ -118,10 +119,14 @@ def render_ctmpl(file_name, output_file_name, tmp_dir=nil)
       cmd_env, 
       "consul-template", 
       "-config=#{$vault_config_path}", 
+      "-log-level=#{consul_log_level}",
+      "-vault-retry-attempts=3",
```

Comment:
This limits consul to retrying failed Vault calls 3x. The default is something like 12, with an exponential bacckoff, which is why consul-template can hang for around 10 minutes when there's an issue in a file.

Reply:
I  like the concept here - let's see in practice if this ever causes any problems where we might want retries at 5, 7 etc.

Reply:
agree. For now, my sense is that the common case is "you haven't deployed <other profile> so the secret isn't there"